BubbleSort/main.cpp

```diff
@@ -19,17 +20,23 @@ void printArray (int dizi[], int elemanSayisi) {
 
 void bubbleSort(int dizi[], int elemanSayisi) {
     int temp,i,j;
-    
+    bool is_swap = false;    //is_swap variable is set to true if any swap between any two elements occur   
     printArray(dizi, elemanSayisi);
     
     for (i = 0; i < elemanSayisi; i++) {
+         is_swap = false;
         for (j = 0; j < elemanSayisi-i-1; j++) {
             if ( dizi[j] > dizi[j+1] ) {
                 temp = dizi[j];
                 dizi[j] = dizi[j+1];
                 dizi[j+1] = temp;
+                 is_swap = true;
             }
         }
+        if (is_swap == false)      //if no swap occurs implies all the elements are in sorted order. No need of conditions to loop to run further. 
+        {
+          break;   
+        }
```

Comment:
I think code is very clear and readable. You can remove comment line or put it to above of if condition. Also "if" parenthesis usage is not same on the program. Put first parenthesis next of if condition and second one is below of if condition. This is not fatal issue but its important for standardization.

Reply:
@muhammedkarakul Done please check.

Reply:
Done 👍